modules/cli/src/main/scala/scala/cli/commands/scalafix/Scalafix.scala

```diff
@@ -0,0 +1,174 @@
+import scala.cli.commands.{ScalaCommand, SpecificationLevel, compile}
+import scala.cli.config.Keys
+import scala.cli.util.ArgHelpers.*
+import scala.cli.util.ConfigDbUtils
+import scala.jdk.CollectionConverters.*
+import scala.jdk.OptionConverters.*
+
+object Scalafix extends ScalaCommand[ScalafixOptions] {
+  override def group: String = HelpCommandGroup.Main.toString
+  override def sharedOptions(options: ScalafixOptions): Option[SharedOptions] = Some(options.shared)
+  override def scalaSpecificationLevel: SpecificationLevel = SpecificationLevel.SHOULD
+
+  val hiddenHelpGroups: Seq[HelpGroup] =
+    Seq(
+      HelpGroup.Scala,
+      HelpGroup.Java,
+      HelpGroup.Dependency,
+      HelpGroup.ScalaJs,
+      HelpGroup.ScalaNative,
+      HelpGroup.CompilationServer,
+      HelpGroup.Debug
+    )
+  override def helpFormat: HelpFormat = super.helpFormat
+    .withHiddenGroups(hiddenHelpGroups)
+    .withHiddenGroupsWhenShowHidden(hiddenHelpGroups)
+    .withPrimaryGroup(HelpGroup.Format)
+  override def names: List[List[String]] = List(
+    List("scalafix")
+  )
+
+  override def runCommand(options: ScalafixOptions, args: RemainingArgs, logger: Logger): Unit = {
+    val buildOptions = buildOptionsOrExit(options)
+    val buildOptionsWithSemanticDb = buildOptions.copy(scalaOptions =
+      buildOptions.scalaOptions.copy(semanticDbOptions =
+        buildOptions.scalaOptions.semanticDbOptions.copy(generateSemanticDbs = Some(true))
+      )
+    )
+    val inputs        = options.shared.inputs(args.all).orExit(logger)
+    val threads       = BuildThreads.create()
+    val compilerMaker = options.shared.compilerMaker(threads).orExit(logger)
+    val configDb      = ConfigDbUtils.configDb.orExit(logger)
+    val actionableDiagnostics =
+      options.shared.logging.verbosityOptions.actions.orElse(
+        configDb.get(Keys.actions).getOrElse(None)
+      )
+
+    val (sourcePaths, workspace, _) =
+      if (args.all.isEmpty)
+        (Seq(os.pwd), os.pwd, None)
+      else {
+        val s = inputs.sourceFiles().collect {
+          case sc: Script          => sc.path
+          case sc: SourceScalaFile => sc.path
+        }
+        (s, inputs.workspace, Some(inputs))
+      }
+
+    val scalaVersion =
+      options.buildOptions.orExit(logger).scalaParams.orExit(logger).map(_.scalaVersion)
+        .getOrElse(Constants.defaultScalaVersion)
+    val configFilePathOpt = options.scalafixConf.map(os.Path(_, os.pwd))
+    val relPaths          = sourcePaths.map(_.toNIO.getFileName)
+    val toolClasspath     = options.shared.dependencies.compileOnlyDependency
+
+    val scalafix = ScalafixInterface
+      .fetchAndClassloadInstance("2.13")
+      .newArguments()
+      .withParsedArguments(options.scalafixArg.asJava)
+      .withWorkingDirectory(workspace.toNIO)
+      .withPaths(relPaths.asJava)
+      .withRules(options.rules.asJava)
+      .withConfig(Optional.ofNullable(configFilePathOpt.map(_.toNIO).orNull))
```

Comment:
with 
```scala
import scala.jdk.OptionConverters.*
```
you can just
```suggestion
      .withConfig(configFilePathOpt.map(_.toNIO).toJava)
```